### PR TITLE
add dbpedia_14 dataset

### PR DESCRIPTION
**Pull request by hfawaz**
This dataset corresponds to the DBpedia dataset requested in https://github.com/huggingface/datasets/issues/353.

### CODE REVIEW

File: datasets/dbpedia_14/dbpedia_14.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+(1 to 14), title and content. The title and content are escaped using double quotes ("), and any
+internal double quote is escaped by 2 double quotes (""). There are no new lines in title or content.
+"""
+
+_HOMEPAGE = "https://wiki.dbpedia.org/develop/datasets"
+
+_LICENSE = "Creative Commons Attribution-ShareAlike 3.0 and the GNU Free Documentation License"
+
+_URLs = {
+    "dbpedia_14": "https://drive.google.com/uc?export=download&id=0Bz8a_Dbh9QhbQ2Vic1kxMmZZQ1k",
```
Comment (lhoestq) [inline datasets/dbpedia_14/dbpedia_14.py:57]: where does the url come from ?

**Review by lhoestq — COMMENTED** [review]
Thanks for adding this one :) looks good to me

Could you add the tags of the dataset card ? It's the yaml part at the top of the dataset card
more info here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

File: datasets/dbpedia_14/dbpedia_14.py (hunk 1 above)
Comment (hfawaz) [reply datasets/dbpedia_14/dbpedia_14.py:57]: I took the same link [here](https://text-docs.readthedocs.io/en/latest/_modules/torchtext/datasets/text_classification.html).

**Review by lhoestq — APPROVED** [review]
Thanks !
LGTM :)

**Comment by hfawaz** [comment]
Thanks for the review. 
Cheers!

**Comment by thomwolf** [comment]
Hi @hfawaz, this week we are doing the 🤗 `datasets` sprint (see some details [here](https://discuss.huggingface.co/t/open-to-the-community-one-week-team-effort-to-reach-v2-0-of-hf-datasets-library/2176)).

Nothing more to do on your side but it means that if you register on the thread I linked above, you can have some goodies for the rpesent dataset that you have already added (and a special goodie if you want to spend more time and add 2 other datasets as well).

If you want to join, just tell me (or post on the thread on the HuggingFace forum: https://discuss.huggingface.co/t/open-to-the-community-one-week-team-effort-to-reach-v2-0-of-hf-datasets-library/2176)

**Comment by hfawaz** [comment]
Hello @thomwolf 
Thanks for the feedback and for this invitation, indeed I would be glad to join you guys (you can add me). 
I will see if I have the time to implement a couple of datasets. 
Cheers!

**Comment by thomwolf** [comment]
@hfawaz invited you to the slack with your uha email.

Check your spam folder if you can't find the invitation :)

**Comment by hfawaz** [comment]
Oh thanks, but can you invite me on my gmail: hassanismailfawaz@gmail.com 
UHA is my old organization, I haven't had the time to update my online profiles yet.
Thank you

**Pull request merged**